lib/securenative/context/hanami_context.rb

```diff
@@ -1,44 +1,42 @@
 # frozen_string_literal: true
 
-module SecureNative
-  class HanamiContext
-    SECURENATIVE_COOKIE = '_sn'
+class HanamiContext
```

Comment:
Can't use module context since we have a class called context and there's a collision and using `SecureNative::Context::Context` seems like bad practice. how about using

```
module SecureNative
module FrameWorkContext

end
end
```
?